Automattic/wp-calypso
# Hosting Metrics: Toggle the expanded row on click

## Proposed Changes

Toggles the expanded details when the row is clicked:


https://github.com/Automattic/wp-calypso/assets/36432/c72febe0-9c8c-4f21-94aa-db8b8b9d4a61

## Testing Instructions

1. Navigate to `/site-monitoring/<site>/web`
2. Click on the row and see it expand.
